### ISSUE TITLE
Author should be in a FOOTER element

### ISSUE THREAD

**Issue by jigarius** [Lullabot/ckeditor_quote#1]
For accessibility, the author name should be in a `FOOTER` element.

**Comment by camppacifictech** [comment]
I made a PR for this: https://github.com/Lullabot/ckeditor_quote/pull/3

**Comment by juampynr** [comment]
Fixed via https://github.com/Lullabot/ckeditor_quote/pull/3
